### DISCUSSION TITLE
Releases/changelog?

### DISCUSSION THREAD

**Discussion by inwardmovement** [radix-ui/website#619] [General]
How can we get notified of releases/new components versions?
It would be great to have an RSS feed at radix-ui.com/docs/primitives/overview/releases

**Comment by benoitgrelard** [comment]
I love that idea!
Do you want to help us with this? I'm not sure how we could implement it from our current setup which is just a plain markdown file edited manually.

**Reply by inwardmovement** [reply]
For now I can't help, just wanted to give the idea 😁

I know some SSG include this by default like [Hugo](https://gohugo.io/templates/rss/).

**Reply by inwardmovement** [reply]
A simple alternative would be to publish releases on GitHub so we could subscribe to them.
